project/libs/publication/publication-module/src/lib/publication.controller.ts

```diff
@@ -84,12 +98,9 @@ export class PublicationController {
     status: HttpStatus.NOT_FOUND,
     description: 'Not found',
   })
-  @Patch()
+  @Patch('')
```

Comment:
Лучше передать id как Param 

```suggestion 
  @Patch('/:id')
```